Generate Crypt Archive broken

The method to generate a crypt archive 

[https://github.com/Chaste/developer_wiki/wiki/Regenerate-Crypt-archive](https://github.com/Chaste/developer_wiki/wiki/Regenerate-Crypt-archive)

is broken as its ctest is too old. The error is 

`[1:40 PM]Run source init.sh
CMake Error at CMakeLists.txt:34 (cmake_minimum_required):
  CMake 3.22.1 or higher is required.  You are running version 3.16.3
-- Configuring incomplete, errors occurred!`

--- a/.github/workflows/regenerate-archive.yml
+++ b/.github/workflows/regenerate-archive.yml
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
 
     container:
-      image: chaste/runner:oldest-boost
+      image: chaste/runner:portability-jammy
       env:
         RUNNER_OFF: 1
       volumes:

